docs/add-on.md

```diff
@@ -49,6 +49,7 @@ Add-ons can enable and disable themselves using the `conditions` key in the mani
 | start_time | The epoch time that activates the current add-on | Integer | No | Yes |
 | end_time | The epoch time that deactivates the current add-on | Integer | No | Yes |
 | javascript | A script file is executed to change the condition status. See below | String | No | Yes | 
+| translation_threshold | The translation threshold to use. By default 0 (all is included) | Number | No | No |
```

Comment:
Should it be the other way around? i.e. default 1.0, given it seems like the most common case, and can be set to 0 as needed.